test/cli/fake-dir.fixture.ts

```diff
@@ -3,11 +3,13 @@ import * as rimraf from 'rimraf';
 import * as fse from 'fs-extra';
 
 export const FAKE_DIR = path.resolve(__dirname, 'tmp');
```

Comment:
why not using `os.tmpdir()`? This way, we should not have to delete after each test, this would be automatically deleted by the OS.

Reply:
Ah, I used that because I've just created the structure in the test folder. I think the tmp dir from the OS will not solve the multiple generations with the tests. Unfortunately I don't think if it's a good idea to use the tmp dir because it's also outside of the project structure and anyone or any program can create anything inside.

Reply:
I would go for `os.tmpdir() + randomUuid` to have tests as simple as possible (and no pollution for managing this testing folder in your tests).

I'm not worried about using a folder outside of the project structure, that is what the generator does :)

As you wish

Reply:
It's not really important for me, I can just add `path.resolve(os.tmpdir(), 'tikui-cli')`, what do you think about this ?

Reply:
yes why not. The most important part for me is not having to think about create/reset/remove for each test

Reply:
Hum I don't understand how your idea with a UUID will solve the problem because the UUID will be generated statically, so is it generated for the whole execution ? for each tests ? for each describe ? for each keyword (it.each) ?